app/controllers/users_controller.rb

```diff
@@ -1,4 +0,0 @@
-class UsersController < ApplicationController
-  def profile
-  end
-end
```

Comment:
helper_method :current_order
delegate :current_order to: :current_user

class User
  def current_order
    order = orders.find_by(state: :cart)
    return order unless order.nil?

    
  end
end